x-pack/test_serverless/functional/test_suites/observability/cypress/e2e/observability_onboarding/home.cy.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+describe('[Serverless Observability onboarding] Landing page', () => {
+  beforeEach(() => {
+    cy.loginAsElasticUser();
+  });
+
+  it('shows landing page', () => {
+    cy.visitKibana('/app/observabilityOnboarding');
+    cy.contains('Get started with Observability');
+  });
+
+  describe('Entry point', () => {
+    it('from Add data', () => {
+      cy.contains('Add data').click();
+      cy.url().should('include', '/app/observabilityOnboarding');
+    });
+  });
+});
+
+export {};
```

Comment:
🙈 Most likely I introduced that in the navigation.cy.ts test but I cannot recall why. Do think is needed?

Reply:
We need to use `"isolatedModules": false` in the tsconfig, I tried  and got rid of it but couldn't make it work for `navigation.cy.ts` 😞 
Do you think would be ok that way? or keep consistency?

Reply:
As discussed, it's fine to leave it like this